nbformat/v4/nbformat.v4.schema.json

```diff
@@ -59,6 +59,14 @@
                     "description": "Original notebook format (major number) before converting the notebook between versions. This should never be written to a file.",
                     "type": "integer",
                     "minimum": 1
+                },
+                "title": {
+                    "description": "The title of the notebook document",
+                    "type": "string"
+                },
+                "author": {
```

Comment:
author probably needs to be a list of authors with a few fields, as described in https://github.com/jupyter/nbformat/issues/45#issuecomment-245572621